test/distributed/_composable/test_composability/test_pp_composability.py

```diff
@@ -537,12 +537,6 @@ def apply_tp(
             else:
                 pipeline_schedule.step()
 
-            # accumulate losses across pipeline microbatches
-            loss = (
```

Comment:
this one is most likely a bug as we should call a `.backward()` on this one.
cc @kwen2501 ?

Reply:
cc., @H-Huang